libraries/kotlin.test/testng/src/test/kotlin/TestNGContributorTest.kt

```diff
@@ -5,24 +5,21 @@ import org.testng.*
 import java.util.concurrent.*
 
 class TestNGContributorTest {
+
     @Test
     fun smokeTest() {
-        Assert.assertEquals("TestNGAsserter", asserter.`class`.simpleName)
+        Assert.assertEquals("TestNGAsserter", kotlin.test.asserter)
```

Comment:
This won't pass most likely, you're comparing `Asserter` object with a String

Reply:
i run `./gradlew kotlin-test:kotlin-test-testng:test` and `./gradlew kotlin-test:kotlin-test-testng:test` before committing, everything was good